tests/node/helpers/build-owner.js

```diff
@@ -1,5 +1,14 @@
 module.exports = function buildOwner(Ember, resolver) {
-  var Owner = Ember.Object.extend(Ember._RegistryProxyMixin, Ember._ContainerProxyMixin);
+  var FACTORY_FOR = Ember.Container.__FACTORY_FOR__;
```

Comment:
See comments in https://github.com/emberjs/ember.js/pull/14360#pullrequestreview-13465262 and https://github.com/emberjs/ember-test-helpers/pull/191, we should be able to remove this.